### PR TITLE
feat(Dropdown): Add property to ignore special chars

### PR DESCRIPTION
**Pull request by MichelAraujo**
## Description
The current version of the component Dropdown when used how autocomplete don't return list if input is not identical, exemple with a itens list how [ 'são paulo',  'belém'] just return if input is 'são paulo' or 'belém'.
With this parameter "ignoreSpecialChars" we can input 'sao paulo' and return list the suggest corrctly.

Changes list:
* Add new parameter in Dropdown component
* Create a function to remove special chars inside the Dropdown component
* Add unit test for Dropdown autocomplete with and without new prop
* Add new parameter in doc storybook 

## Review guide
- [ ] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [x] Unit tests (yarn test:components)
- [ ] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [ ] Code review

### Browsers review
- [ ] Layout review
  - [x] Edge
  - [x] IE 11
  - [ ] Firefox
  - [x] Chrome
  - [ ] Safari
  - [x] Mobile
- [ ] Ux review validation

### CODE REVIEW

File: components/Dropdown/Dropdown.jsx [hunk 1]
```diff
@@ -325,14 +326,31 @@ const Dropdown = ({
 
   const [_id] = useState(id || ID_GENERATOR.next().value);
 
-  const inputFilter = value =>
-    items.filter(
-      item =>
-        _getValue(item)
-          .toLowerCase()
-          .indexOf(value.toLowerCase()) > -1,
+  const normalizeChars = value =>
```
Comment (renatogalvones) [inline components/Dropdown/Dropdown.jsx]: Not a block, but would be nice to have this function isolated so we can reuse it on other components.

Suggestion: `Components/shared/` folder. 

Just like `hexToRgba`:

https://github.com/catho/quantum/blob/master/components/shared/hexToRgba.js

Comment (MichelAraujo) [reply components/Dropdown/Dropdown.jsx]: great idea!
Done!

Comment (renatogalvones) [reply components/Dropdown/Dropdown.jsx]: :tada:

**Pull request merged**